lib/process-release.js

```diff
@@ -0,0 +1,87 @@
+var semver = require('semver')
+var url = require('url')
+var path = require('path')
+
+var bitsre = /(x32|x64)/
+
+function processRelease (argv, gyp, defaultVersion, defaultRelease) {
+  var versionStr = argv[0] || gyp.opts.target || defaultVersion
+
+  // parse the version to normalize and ensure it's valid
+  var versionSemver = semver.parse(versionStr)
```

Comment:
done, but left the variable as `versionSemver`, exposing it as a `semver` property on the return object, remember we have a `semver` package required in here